frontend/src/modules/controllers/ValueClicker.tsx

```diff
@@ -89,6 +133,13 @@ export const ValueClicker = ({
     }, disabledInterval);
   };
 
+  /**
+   * Button Click Wrapper
```

Comment:
What does this wrapper do?

Reply:
It calls Click handler based on Increment/Decrement and also it calls `setActiveRotaryReference` function which updates the red border around the `ValueClicker` UI wrapper